charts/base-cluster/README.md

```diff
@@ -214,6 +214,10 @@ This helm chart requires flux v2 to be installed (https://fluxcd.io/docs/install
 | speedtest.image.registry | string | `""` |  |
 | speedtest.image.repository | string | `"adolfintel/speedtest"` |  |
 | speedtest.image.tag | string | `"latest"` |  |
+| traefik.debug.enabled | bool | `true` |  |
+| traefik.image.tag | string | `"v2.11.2"` |  |
```

Comment:
This is probably not the best idea. Before this change the tag was managed through the referenced traefik helm chart. After this commit, we have to take care of this ourselves, which is IMHO not acceptable.

Reply:
Changed